src/get-started/java-binding.md

```diff
@@ -222,6 +222,26 @@ reference. Light clients also provide access to information on the
 [transaction][exonum-transaction] execution result
 (which may be either success or failure) to their users.
 
+### Blockchain Events
+
+A service can also handle a block commit event that occurs each time
```

Comment:
```suggestion
A service may handle a block commit event that occurs each time
```

Reply:
I'd keep can, as it seems to fit here better (describing a capability)